codingTest_김준우/신규 아이디 추천.js

```diff
@@ -0,0 +1,54 @@
+// 아이디 추천하는 프로그램
+// 아이디 길이 3~15자 이하
+// 알파벳 소문자 / 숫자 / 빼기(-) / 밑줄(_) / 마침표(.)
+// 마침표(.)는 처음과 끝에 사용할 수 없으며 연속으로 안된다.
+
+function solution(new_id) {
+    let answer = "";
+    answer = new_id.toLowerCase();
+    const regex = /[^a-z0-9\-\_\.]+/g;
+    if (regex.test(answer)) {
+      answer = answer.replace(regex, "");
+    }
+    console.log(answer)
+    const regex1 = /[\.]{2,}/g;
+    if (regex1.test(answer)) {
+      answer = answer.replace(regex1, ".");
+    }
+    const regex2 = /^\./;
+    const regex3 = /\.$/;
+    if (regex2.test(answer) || regex3.test(answer)) {
+      answer = answer.replace(regex2, "");
+      answer = answer.replace(regex3, "");
+    } console.log(answer)
```

Comment:
```suggestion
    } 
```
의미없는 console.log는 삭제!